src/api/app/models/owner.rb

```diff
@@ -101,8 +101,10 @@ def self.find_assignees(rootproject, binary_name, limit = 1, devel = true, filte
       data.elements('binary').each do |b| # no order
         next unless b['project'] == prj.name
 
-        pkg = prj.packages.find_by_name(b['package'])
-        next if pkg.nil?
+        package_name = b['package']
+        package_name.gsub!(/\.[^\.]*$/, '') if prj.is_maintenance_release?
```

Comment:
Am I right that this would transform e.g. ``aaa_base.import4171`` -> ``aaa_base``?
I would prefer then ``package_name.split('.').first`` as the regex is quite hard to understand.
Also I would like to get this assigned to a proper name e.g. ``base_name`` (maybe you have a better suggestion?). Then we could also move this as a function to package and call it always.

```pkg = prj.packages.find_by_name(Package.base_name(b['package']))```

Reply:
split('.') would break if you have more than one dot. IMHO this is really a very simple regexp and we use it all over the place. you may want to refactor it into a generic method though.